compiler/rustc_smir/src/rustc_smir/context.rs

```diff
@@ -587,6 +587,13 @@ impl<'tcx> Context for TablesWrapper<'tcx> {
         }
     }
 
+    fn intrinsic_name(&self, def: InstanceDef) -> Option<Symbol> {
+        let tables = self.0.borrow_mut();
+        let instance = tables.instances[def];
+        let name = tables.tcx.intrinsic(instance.def_id())?;
+        Some(name.to_string())
```

Comment:
```suggestion
        let intrinsic = tables.tcx.intrinsic(instance.def_id())?;
        Some(intrinsic.name)
```

Reply:
Renamed. Doesn't look like I can avoid the `to_string()` call though.